python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 1.5.0
+
+### Feature:
+- Support Microsoft SQL as Database [#1538](https://github.com/astronomer/astro-sdk/pull/1538)
+- Support DuckDB as Database. `aql.merge` requires a fix on DuckDB as `UPDATE/IGNORE` clause is not supported as per [bug on duckdb](https://github.com/duckdb/duckdb/issues/6087) [#1695](https://github.com/astronomer/astro-sdk/pull/1695)
+- Add `load_options` param to accept `pandasOptions` in `load_file` operator [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
```

Comment:
why we are including this in 1.5.0? We have already released this in 1.4.0 https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/CHANGELOG.md#140 cc: @phanikumv

Reply:
Its part of 1.5.0a1 https://github.com/astronomer/astro-sdk/pull/1666

Reply:
[Removed](https://github.com/astronomer/astro-sdk/pull/1714/commits/14ec0a8ce30619c465cc4a3981e734b542b65db4) it as this was released and this issue uses this feature: https://github.com/astronomer/astro-sdk/issues/1264